src/components/EntityPage/File/fileProperties.js

```diff
@@ -1,74 +1,325 @@
 import React from 'react';
 import uniq from 'lodash/uniq';
-import {pickData} from './utils';
-import {formatBytesToHumanReadable} from './utils';
-import {trackUserInteraction, TRACKING_EVENTS} from 'services/analyticsTracking';
-import {kfWebRoot} from 'common/injectGlobals';
+import { pickData } from './utils';
+import { formatBytesToHumanReadable } from './utils';
+import { trackUserInteraction, TRACKING_EVENTS } from 'services/analyticsTracking';
+import { kfWebRoot } from 'common/injectGlobals';
 
 import ExternalLink from 'uikit/ExternalLink';
+import { DB_GA_P, generateUrlForDbGap } from 'common/constants';
 
-const buildParticipantsTableConfigs = participants => {
-    if (!participants) {
-        return [];
+export const fileQuery = `query ($sqon: JSON) {
```

Comment:
I'm just wandering if opposed to building this giant query you ca just use the `fetchData` in Arranger.js (Arranger). In turn this uses the `columnsToGraphql` from mapping-utils. You would just need to pass the field or accessors as required..